src/main/resources/xml/xproc/step/2020-1/epub3-validate.step.xpl

```diff
@@ -253,7 +253,7 @@
                     <p:pipe port="in-memory" step="epub3-validate.step.unzip"/>
                 </p:input>
             </px:fileset-load>
-            <px:assert test-count-min="1" test-count-max="1" message="There is no NCX with the filename 'nav.ncx' in the EPUB" error-code="NORDICDTBOOKEPUB014"/>
+            <px:assert test-count-min="0" test-count-max="1" message="There is no NCX with the filename 'nav.ncx' in the EPUB" error-code="NORDICDTBOOKEPUB014"/>
```

Comment:
The error text is a bit misleading now that the NCX is optional, and I think the test will in practice always succeed anyway, because of the `<p:delete match="/*/*[not(ends-with(@href,'nav.ncx'))]"`. I guess it would catch cases where there are multiple files called `nav.ncx` but in different directories. Validation of whether or not there are multiple NCX files, and whether or not they have the correct file name is something that is better to do in the opf schematron validation anyway, so I think this is fine as it is.